README.md

```diff
@@ -226,6 +231,21 @@ WHERE rds.storage_encrypted = false
 return a.name as AWSAccount, count(rds) as UnencryptedInstances
 ```
 
+### What users have the Grammarly for Chrome extension installed?
+```
+MATCH (u:GSuiteUser)-[r:INSTALLS]->(ext:ChromeExtension)
+WHERE ext.name CONTAINS 'Grammarly'
+return ext.name, ext.version, u.email
+```
+
+### What users have installed extensions that are risky based on [CRXcavator scoring](https://crxcavator.io/docs#/risk_breakdown)?
```

Comment:
Use `####` not `###` to match indent level with the other example questions in this section.

Reply:
fixed